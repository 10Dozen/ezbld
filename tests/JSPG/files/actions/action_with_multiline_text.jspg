@ Action with multiline text | scene_right | MyTag

*scene: CurrentScene
*goto: "Scene2"

Block 1.
Line 1.
Line 2.

Block 2.
Line ${1}.
Line ${2}.

Line ${3}