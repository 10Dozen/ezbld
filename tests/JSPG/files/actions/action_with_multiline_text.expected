[0]
cls = JSPGAction
name = Action with multiline text | TEXT
type = scene_right | TEXT
tag = MyTag | TEXT
scene = CurrentScene | TEXT
goto = Scene2 | TEXT
desc =
    `Block 1.<br>
    Line 1.<br>
    Line 2.<br>`

    `>>> Block 2.<br>
    Line \${1}.<br>
    Line \${2}.<br>`

    "`Line ${3}`"
